vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -454,7 +455,14 @@ export default class SbcHeader extends Mixins(NavigationMixin) {
   }
 
   private goToCreateBCSCAccount () {
-    this.redirectToPath(this.inAuth, Pages.CREATE_ACCOUNT)
+    if (this.inAuth) {
```

Comment:
use the existing function this.redirectToPath if we can re-use

Reply:
Thanks, I have refactored the changes